src/nvim/api/ui.c

```diff
@@ -700,3 +741,31 @@ static void remote_ui_inspect(UI *ui, Dictionary *info)
   UIData *data = ui->data;
   PUT(*info, "chan", INTEGER_OBJ((Integer)data->channel_id));
 }
+
+void redraw(uint64_t channel_id, Array uidata, Error *error)
```

Comment:
where is this used? Would be good to mention it in a brief comment, e.g. 

    // see gen_api_ui_events.lua

Reply:
Name is too generic, we should probably mention "ui client" in all ui code running in the client, which is not specific to TUI (i e at a later stage could form basis for UI support in libnvim).